draft-jhoyla-req-mtls-flag.md

```diff
@@ -31,22 +31,22 @@ informative:
 
 --- abstract
 
-Normally in TLS there is no way for the client to signal to the server that it
-supports client authentication and will handle a `CertificateRequest`
-gracefully. This document defines a TLS Flag that enables clients to provide
-this hint.
-
+This document defines a TLS Flag that allows a TLS client to signal to the
+server that it is a bot. This allows the server to then request a client
+certificate without risking the poor UI/UX that users can experience when
+connecting to a zone that requests a client certificate.
```

Comment:
I like the abstract as is. Perhaps just tack on a sentence that says something like "the primary use case for this mechanism is ..."